packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -811,6 +812,18 @@ class RawScrollbarState<T extends RawScrollbar> extends State<T> with TickerProv
   @override
   void didChangeDependencies() {
     super.didChangeDependencies();
+    switch (defaultTargetPlatform) {
+      case TargetPlatform.android:
+      case TargetPlatform.iOS:
+        _isMobile = true;
+        break;
+      case TargetPlatform.fuchsia:
```

Comment:
We can add a default branch, in case a new platform is added in the future but not syncing will trigger a late value error.